src/App.tsx

```diff
@@ -52,7 +52,7 @@ export const PermissionContext = createContext<{
   isOrgAdmin?: boolean;
 }>({});
 
-const App = () => {
+const App = ({ layoutType } : {layoutType?: string }) => {
```

Comment:
It's better to write Ract.FC with props
```suggestion
const App: React.FC<{layoutType?: string }> = ({ layoutType }) => {
```